settings.gradle.kts

```diff
@@ -17,7 +17,10 @@ include("agentextension")
 include("bootstrap")
 include("custom")
 include("instrumentation")
-include("instrumentation:openai-client-instrumentation")
+include("instrumentation:openai-client-instrumentation:common")
+include("instrumentation:openai-client-instrumentation:testing-common")
+include("instrumentation:openai-client-instrumentation:instrumentation-0.13.0")
+include("instrumentation:openai-client-instrumentation:instrumentation-latest")
```

Comment:
[question] In the upstream instrumentation, I think the recommended pattern is to always include the minimum version supported for a module, both in the gradle module name and in the code packages. There is this draft PR upstream: https://github.com/open-telemetry/opentelemetry-java-instrumentation/pull/12965 so maybe starting to use this for our own instrumentations would be a good start as it's less painful than changing it later.